### PR TITLE
Change TTL for polyfills to 1 week

### PR DESCRIPTION
**Pull request by richard-still-ft**

### CODE REVIEW

File: service/index.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ var contentTypes = {".js": 'application/javascript', ".css": 'text/css'};
 
 // Default cache control policy
 app.use(function(req, res, next) {
-	res.set('Cache-Control', 'public, max-age='+one_day+', stale-while-revalidate='+one_week+', stale-if-error='+one_week);
+	res.set('Cache-Control', 'public, max-age='+one_week+', stale-while-revalidate='+one_week+', stale-if-error='+one_week);
```
Comment (matthew-andrews) [inline service/index.js:32]: the stale bits don't really do anything anymore :relaxed:

Comment (triblondon) [inline service/index.js:32]: Hey, hold on a minute. Read the spec!

image

Comment (matthew-andrews) [inline service/index.js:32]: oh wow ok , sorry for the confusion @richard-still-ft  …

Comment (richard-still-ft) [inline service/index.js:32]: I should have read the spec!

**Comment by samgiles** [comment]
There's no harm in increasing the `stale-while*` values though if we're willing to increase default `max-age` to a week. Previously `stale-while*` values were 7x higher. :information_desk_person:

**Pull request merged**